mpp/mpp.cpp

```diff
@@ -626,6 +638,9 @@ MPP_RET Mpp::control_mpp(MpiCmd cmd, MppParam param)
         MppPollType block = *((MppPollType *)param);
         mOutputBlock = block;
     } break;
+    case MPP_SET_OUTPUT_BLOCK_TIMEOUT: {
+        mOutputBlockTimeout = *((int *)param);
```

Comment:
The mOutputBlockTimeout is 64bit but *param will be 32bit. Although the value range is enough but it is better to get a 64bit value here?